perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -160,21 +186,25 @@ def score_subreddit(creds_dict,
                        username=creds_dict['reddit_username'],
                        password=creds_dict['reddit_password'])
   subreddit = reddit.subreddit(subreddit_name)
-  mod_permissions = bot_is_mod(reddit, subreddit)
+  original_mod_permissions = bot_is_mod(reddit, subreddit)
 
-  if mod_permissions:
+  if original_mod_permissions:
     print('Bot is moderator of subreddit.')
     print('Moderation actions will be applied.')
   else:
     print('Bot is not moderator of subreddit.')
     print('Moderation actions will not be applied.')
 
-  perspective = perspective_client.PerspectiveClient(creds_dict['perspective_api_key'])
+  perspective = perspective_client.PerspectiveClient(
+    creds_dict['perspective_api_key'])
 
   for i, comment in enumerate(subreddit.stream.comments()):
     try:
```

Comment:
I think it would be good to pull the inside of this loop into a function at some point :) add a TODO?

Reply:
Definitely agreed. I've added a TODO for the next refactor.